custom/abdm/milestone_one/utils/abha_creation_util.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+from custom.abdm.milestone_one.utils.request_util import get_response_http_post
+
+logger = logging.getLogger(__name__)
```

Comment:
this seems redundant now? Its note being used anywhere

Reply:
https://github.com/dimagi/commcare-hq/pull/32332/commits/bc17f6ed29bc577b651d5d9991c71643a10f5600